GMG/Scenes/ChordProgress/ChordProgressView.swift

```diff
@@ -459,33 +594,32 @@ extension ChordProgressView {
                 GeometryReader { proxy in
                     let totalSpacing = Spacing.xs * CGFloat(chordCellsWithDuration.count - 1)
                     let availableWidth = proxy.size.width - totalSpacing
-                    
+
                     HStack(spacing: Spacing.xs) {
                         ForEach(chordCellsWithDuration, id: \.0) { (chordCell, duration) in
                             let widthRatio = duration / max(1, segmentDuration)
-                            let cellWidth = max(1, availableWidth * widthRatio)
-                            
+                            let cellWidth = availableWidth * widthRatio
+                            let clampedWidth = cellWidth < 1 ? 0 : cellWidth
+
                             ZStack {
                                 if let chord = chordCell.chord {
                                     ChordCellButton(
                                         chord: chord,
-                                        isCurrentChord: currentChordCell?
-                                            .startTime
+                                        showChordDescription: widthRatio >= 0.2,
```

Comment:
0.02인데 오타난 것 같아요!

<p align="center">
  <img width="256" alt="ChordProgressView Segment" src="https://github.com/user-attachments/assets/69b8e016-39b5-40b3-a4a7-74c48d4e7d71" />
</p>

```suggestion
                                        showChordDescription: widthRatio >= 0.02,
```

Reply:
이거는 텍스트가 보이는지 말지에 대한 비율이라 20%가 맞습니다! 아리랑 이야기된 부분이에요!

Reply:
다소 헷갈리는 부분이 있을 수 있으니 코드를 좀 수정해볼게용